tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Devices and Virtual Devices"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from dataclasses import replace\n",
+    "from pulser.devices import Device, VirtualDevice, Chadoq2, MockDevice\n",
+    "from pulser.channels import Rydberg, Raman, Microwave\n",
+    "from pulser import Pulse, Sequence, Register"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Physical devices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform any computation using Pulser, it is necessary to define a device. Some physical devices are available, they can be accessed via `pulser.devices`. These devices are instances of the `Device` class. They are constrained by physical considerations and all their parameters are defined. Below is presented the specifications of the physical device `Chadoq2`, that are accessible via the `Device.print_specs()` method. "
```

Comment:
Here are a few suggestions, feel free to adopt the ones you agree with:
```suggestion
    "To perform any computation using Pulser, it is necessary to choose a device. For convenience, some examples of typical physical devices are included and can be accessed via `pulser.devices`.\n
     These devices are instances of the `Device` class. By definition, a `Device` represents a physical device and, as such, requires that all its parameters are defined. \n
     As an example, we present below the specifications of the physical device `Chadoq2`, which can be accessed via the `Device.print_specs()` method. "
```

Reply:
By the way, the formatting on these suggestions is probably screwed up, so don't just commit them even if you agree with everything